Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,239 @@
 /*
-Project Name: HTML Template
-Client: Your Client
-Author: Your Name
-Developer: @poopsplat in Chicago
+Project Name: Landing Page
+Client: Relaxr
+Author: Haley 
+Developer: @needleinthehaley in Chicago
 */
 
 /******************************************
-/* BASE STYLES
+/* UTILITY
 /*******************************************/
-body {
-  color: black;
-  font-size: 16px;
-  background: white;
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+}
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
 }
 
+/*i started off with ALL intentions of utilizing the categories 
+outlined in the CSS template. that is SO hard to do when you are
+constantly trying to figure everything out. i didn't do it this 
+time, but if i have time i'll go back in and sort. i'm sure 
+duplication of code is annoying to review.*/
 /******************************************
-/* LAYOUT
+/* BASE STYLES
 /*******************************************/
+body {
+	color: black;
+	margin: 0;
+	font-size: 16px;
+	font-family: 'open sans', sans-serif;
+	width: 100%;
+}
 
-/******************************************
-/* PAGES
-/*******************************************/
+/****************************************/
+/*it looks like there is an opacity piece on top of my image, 
+but i can't figure it out! why is it so hazy?*/
```

Comment:
Hmm, looks fine to me! Might just be when you view it at a size that it gets stretched that it looks hazy?